### ISSUE TITLE
Change vendor/product ids for different headset?

### ISSUE THREAD

**Issue by AnhadSodhi** [franco-giordano/cloud-flight-monitor#5]
Hi, I've been looking for a battery monitor for my HyperX Cloud Alpha Wireless headset, and this project fits what I'm looking for. I can't find one specifically for my headset, however. I'm not familiar with C#, is there a simple way to edit the vendor and product ids for my headset? The vendor id is 03F0 and the product id is 098D. I'm fine with doing it myself if you point me to which file those are stored in, if it's as simple as changing a couple values.

**Comment by Jason-THL** [comment]
Hey, did you figure out how to do this?

**Comment by AnhadSodhi** [comment]
No, I ended up using csmith's alphabat (https://github.com/csmith/alphabat) and wrote a little batch script to run it from my desktop whenever I need to check. It takes a few seconds to start up, but works perfectly as far as I can tell.
